File: apps/mobile/ios/Runner/Native/Camera/FocusRingConfig.swift
import UIKit

enum FocusRingConfig {
  static let color: UIColor = .white
  static let strokeWidth: CGFloat = 1.5
  static let duration: CFTimeInterval = 1.2
  static let scaleFrom: CGFloat = 1.4
  static let scaleTo: CGFloat = 1.0
  static let radius: CGFloat = 32.0
  static let opacityKeyframes: [NSNumber] = [0.0, 1.0, 0.0]
  static let opacityKeyTimes: [NSNumber] = [0.0, 0.2, 1.0]
}

File: packages/shared/lib/src/camera/focus_ring_config.dart
class FocusRingConfig {
  const FocusRingConfig._();

  static const int colorArgb = 0xFFFFFFFF;
  static const double strokeWidth = 1.5;
  static const int durationMs = 1200;
  static const double scaleFrom = 1.4;
  static const double scaleTo = 1.0;
  static const double radiusPx = 32.0;
  static const List<double> opacityKeyframes = [0.0, 1.0, 0.0];
  static const List<double> opacityKeyTimes = [0.0, 0.2, 1.0];
}
